### ISSUE TITLE
Rename sketch theme to LowFidelity theme

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#1302]
Rename the "sketch" theme to "LowFidelity" to match terminology on the New HTML File dialog. Should be straightforward, right? Just rename files and change "sketch" to "LowFidelity" after doing a grep. 

@waynevicknair: we will need to deal with this somehow in the migration scripts.

**Comment by JonFerraiolo** [comment]
This one will be put on hold temporarily why we work through terminology issues on the New HTML File dialog.

**Comment by JonFerraiolo** [comment]
Pushing to M5. Not critical for M4 and risky, particularly on the migration front issue. Also, small chance current terminology (low fidelity vs high fidelity) might change again.

**Comment by JonFerraiolo** [comment]
@billreed63 
Moving to M6 when big theme editor changes will be happening.

**Comment by JonFerraiolo** [comment]
Seems unlikely the big theme editor changes will happen in time for M6, so will probably push this to M7, but leaving as M6/High for time being.

**Comment by JonFerraiolo** [comment]
Pushing to M7 - leaving open in case we still want to do theme renaming, but still waiting on our terminology to solidify

**Comment by JonFerraiolo** [comment]
-> M8
